src/gboml/scope.py

```diff
@@ -175,7 +212,7 @@ def __post_init__(self):
     def _finalize_init(self):
         if intersection := self.parent.content.keys() & self.ast.args:
             raise RuntimeError(f"Identifier {intersection} is already used")
-        self.content = self.parent.content | dict.fromkeys(self.ast.args, {})
+        self.content |= dict.fromkeys(self.ast.args, {})
```

Comment:
Should be changed to use `_add_unnamed_to_scope()` (remember to copy content before)

Reply:
In the current way it's done, this is incorrect because it modifies self.content which is actually self.parent.content, so now the parent has obtained the parameters of the function in its own scope...

Reply:
test

Reply:
In the current way it's done, this is incorrect because it modifies self.content which is actually self.parent.content, so now the parent has obtained the parameters of the function in its own scope...

Reply:
test

Reply:
test